docs/install/source-step-by-step.md

```diff
@@ -0,0 +1,343 @@
+```
+
+````{admonition} mkwsgiinstance's minimal Zope configuration
+:class: margin toggle
+`mkwsgiinstance` creates a home with a minimal configuration for a Zope instance.
+No blob storage, etc. is defined so far.
+
+```
+┌── etc/
+│   ├── site.zcml
+│   ├── zope.conf
+│   └── zope.ini
+├── inituser
+└── var/
+    ├── cache/
+    ├── log/
+    └── REAMDME.txt/
+```
+````
+
+Create a Zope instance with the given username and password in the current directory.
+
+(install-source-mkwsgiinstance)=
+
+```shell
+mkwsgiinstance -u admin:admin -d .
+```
+
+````{admonition} All commands so far
+:class: margin
+
+```shell
+python3.9 -m venv venv
+source venv/bin/activate
+pip install -U pip wheel
+pip install Plone -c https://dist.plone.org/release/6.0.0a4/constraints.txt
+mkwsgiinstance -u admin:admin -d .
+```
+````
+
+Start the Zope instance.
+
+```shell
+runwsgi ./etc/zope.ini
+```
+
+You can stop the instance later with {kbd}`ctrl-esc`.
+
+If you now open the browser with http://localhost:8080/, you see that you already can create a Plone instance.
+Before doing this, we configure our Zope instance for blobs, configure add-ons, etc..
+
+For the configuration, you have two options:
+1. *manual* configuration by editing {file}`site.zcml` and {file}`zope.conf` (^[Configuring and Running Zope](https://zope.readthedocs.io/en/latest/operation.html))
+2. *generate* configuration by applying `cookiecutter-zope-instance`
+
+(install-source-cookiecutter-zope-instance-label)=
+
+#### Generate Plone / Zope configuration with cookiecutter
+
+{term}`Cookiecutter` creates projects from project templates.
+{term}`cookiecutter-zope-instance` is such a template that allows to create a complete Zope configuration. 
+Zope configuration means:
+
+- type of storage: direct filestorage/blobs, relational database, ZEO, you name it.
+- ports, 
+- threads/caches,
+- debugging/profiling options, 
+- ...
+
+Install cookiecutter:
+
+```shell
+pip install cookiecutter
+```
+
+You could now run `cookiecutter` to create a Zope instance sceleton including configuration with the following command. It prompts you for parameter values.
+
+```shell
+cookiecutter https://github.com/plone/cookiecutter-zope-instance
+```
+
+(install-source-cookiecutter-zope-instance-presets-label)=
+
+Instead we prepare a file {file}`instance.yaml` with the parameters we want to set. 
+A minimal example with one add-on configured is (add options as needed):
+
+```yaml
+default_context:
+    initial_user_name: 'admin'
+    initial_user_password: 'admin'
+
+    load_zcml:
+        package_includes: ['collective.easyform']
+
+    db_storage: direct
+```
+
+Find more [options of cookiecutter `cookiecutter-zope-instance`](https://github.com/plone/cookiecutter-zope-instance#options).
+
+The file {file}`instance.yaml` allows to set some presets.
+Add-ons are listed here, but need to be installed with pip.
+The documented installation of add-ons with pip is achieved via a {file}`requirements.txt` file.
```

Comment:
if add-ons are listed here in instance.yaml but need to be installed with pip using a requirements.txt file, why would we need to list them? An explanation should be added.

Reply:
yes, thanks. I rephrased it.